extensions/preset-firstrun.sh

```diff
@@ -0,0 +1,52 @@
+function post_family_tweaks__preset_configs() {
+	display_alert "$BOARD" "preset configs for rootfs" "info"
+	# Set FR_net_change_defaults to 1 to apply any network related settings below
+	echo "FR_net_change_defaults=1" > "${SDCARD}"/root/.not_logged_in_yet
```

Comment:
Perhaps unifying variables?

FR_net_change_defaults -> PRESET_NET_CHANGE_DEFAULTS
FR_net_wifi_ssid -> PRESET_NET_WIFI_SSID

And add somewhere example of manually generated /root/.not_logged_in_yet ... which could also be renamed at this point to .preset_firstrun so its not that "hacky".